draft-ietf-quic-http.md

```diff
@@ -2159,6 +2159,12 @@ the settings identifier space in HTTP/3 is substantially larger (62 bits versus
 16 bits), so many HTTP/3 settings have no equivalent HTTP/2 code point. See
 {{iana-settings}}.
 
+An endpoint SHOULD NOT wait for the peer's settings to arrive before responding
+to other streams, as it cannot assume the peer's settings to arrive in a timely
+manner.  This is because the packet carrying the settings can be lost, or the
```

Comment:
The problem here is that if both endpoints wait for settings from a peer, we have a situation where SETTINGS never gets sent.  If both peers decide to wait, then they wait forever.

Sorry, I wasn't clear that this was the concern in my other comment.

Reply:
Thank you for the clarification.

That's a good point, I agree that it's worth clarifying, so done in 4664b5c. I've added that to the main section, as that is the property that has *not* changed from HTTP/2.